### PR TITLE
feat: add bitgo signet for btc

### PR DESCRIPTION
**Pull request by ericli-bitgo**
TICKET: BTC-1386

### CODE REVIEW

File: modules/statics/src/networks.ts [hunk 1]
```diff
@@ -297,6 +297,13 @@ class BitcoinPublicSignet extends Testnet implements UtxoNetwork {
   explorerUrl = 'https://mempool.space/signet/tx/';
 }
 
+class BitcoinBitGoSignet extends Testnet implements UtxoNetwork {
+  name = 'BitcoinBitGoSignet';
+  family = CoinFamily.BTC;
+  utxolibName = 'bitcoinBitGoSignet';
+  explorerUrl = 'https://mempool.space/notanetwork/tx/';
```
Comment (lzjluzijie) [inline modules/statics/src/networks.ts:304]: So we don't have a explorer for BitGo Signet?

Comment (ericli-bitgo) [reply modules/statics/src/networks.ts:304]: Not right now

File: modules/utxo-lib/test/integration_local_rpc/generate/fixtures.ts [hunk 2]
```diff
@@ -25,8 +25,11 @@ export function getProtocolVersions(network: Network): number[] {
 }
 
 export function getFixtureDir(protocol: Protocol): string {
-  const networkName = getNetworkName(protocol.network);
+  let networkName = getNetworkName(protocol.network);
   assert(networkName);
+  if (networkName === 'bitcoinBitGoSignet') {
```
Comment (zahin-mohammad) [inline modules/utxo-lib/test/integration_local_rpc/generate/fixtures.ts:30]: Wouldn't a coin name of `tbtcbg` make more sense if the network is written like that?

**Review by lcovar — APPROVED** [review]

**Comment by lcovar** [comment]
failing docker build step but guess it's not required @ericli-bitgo

**Comment by ericli-bitgo** [comment]
Build step passing after re-run

**Pull request merged**